e2e-tests/cypress/tests/integration/channels/accessibility/accessibility_post_spec.js

```diff
@@ -18,7 +18,6 @@ describe('Verify Accessibility Support in Post', () => {
     let otherUser;
     let testTeam;
     let testChannel;
-    let emojiPickerEnabled;
```

Comment:
This one for conditional testing here is unreachable since by default, emoji picker is enabled. Such other test case has to be addressed separately.